nsv13/code/controllers/subsystem/processing/overmap.dm

```diff
@@ -5,3 +5,4 @@ PROCESSING_SUBSYSTEM_DEF(overmap)
 	wait = 2
 	stat_tag = "OP"
 	priority = FIRE_PRIORITY_OVERMAP
+	var/next_boarding_time = 0
```

Comment:
Tell me this doesn't make boarding happen immediately.

Reply:
Intentional. But this only happens when large enemies get up close so it's already rare.

Reply:
Should be a warning and a minimum of ten seconds.